api/src/main/scala/hmda/api/http/institutions/SubmissionPaths.scala

```diff
@@ -141,4 +141,39 @@ trait SubmissionPaths
       }
 
     }
+
+  def submissionSingleEditPath(institutionId: String) =
```

Comment:
This endpoint should be a  `GET`, right? In that case, you'll need to use the `get` directive somewhere in the endpoint (or preferably `timedGet`)